package/yast2.spec

```diff
@@ -52,6 +52,10 @@ BuildRequires:  yast2-devtools >= 3.1.10
 # Pkg.Resolvables() with "path" search support
 BuildRequires:  yast2-pkg-bindings >= 4.3.7
 BuildRequires:  rubygem(%rb_default_ruby_abi:yast-rake)
+# ProductEvaluation.rb
+BuildRequires:  yast2-packager
+# RootPart
+BuildRequires:  yast2-update
```

Comment:
Unfortunately this would create a circular dependencies (`yast2` -> `yast2-update` -> `yast2`, the same with `yast2-packager`).

It needs to be implemented differently, the `yast2-packager` and the `yast2-update` code needs to register the callbacks in `yast2`, then we could keep the current dependencies without adding new ones.